src/__tests__/to-be-visible.js

```diff
@@ -32,4 +40,9 @@ test('.toBeVisible', () => {
   expect(() =>
     expect(container.querySelector('p')).toBeVisible(),
   ).toThrowError()
+
+  expect(queryByTestId('visible-details').querySelector('div')).toBeVisible()
+  // expect(queryByTestId('visible-details').querySelector('summary')).toBeVisible()
+  // expect(queryByTestId('hidden-details').querySelector('summary')).toBeVisible()
```

Comment:
Either use these or remove them.

Reply:
Sorry, it'll be removed. The PR is in wip yet.